### DISCUSSION TITLE
mspider/intro/app-spec

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#2891] [Announcements]
# mspider/intro/app-spec

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/mspider/intro/app-spec.html

**Comment by kebe7jun** [comment]
增加 version 标签说明。

**Comment by hanxiaop** [comment]
```
Istio 升级
边车升级
```
不是超链接

**Comment by hanxiaop** [comment]
可以把端口监听部分的端口展开说说
